src/SIL.LCModel.Core/Phonology/phonenv.parser.cs

```diff
@@ -2,6 +2,8 @@
 // This software is licensed under the LGPL, version 2.1 or later
 // (http://www.gnu.org/licenses/lgpl-2.1.html)
 
+#nullable enable
```

Comment:
yeah maybe let's not enable it here since we just had to put `!` all over the place, I'm not sure it's adding anything